designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,240 @@
+```
+Once this spec has been implemented the output of `nuget.exe verify` command for various verbosity levels will change and be in sync with `dotnet nuget verify` command.
+```
+
+### Log level mapping - The following details are copied from [here](https://github.com/NuGet/Home/blob/dev/designs/Package-List-Verbosity.md#log-level-mapping). Thanks to [Joel Verhagen](https://github.com/joelverhagen) for the detailed information.
+
+The provided `--verbosity` value will include to NuGet log messages with the following levels:
+
+​             | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etails]` | `diag[nostic]`
+------------- | --------- | ----------- | ---------- | ----------- | --------------
+`Error`       | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Warning`     | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Minimal`     | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+`Information` | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+`Verbose`     | ❌        | ❌         | ❌         | ✔️         | ✔️   
+`Debug`       | ❌        | ❌         | ❌         | ✔️         | ✔️   
+
+Note that MSBuild itself has the following mapping for it's own "log levels"
+([source](https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019#verbosity-settings)):
+
+​                                     | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
+------------------------------------- | --------- | ----------- | ---------- | ----------- | --------------
+Errors                                | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+Warnings                              | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+High-importance Messages              | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+Normal-importance Messages            | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+Low-importance Messages               | ❌        | ❌         | ❌         | ✔️         | ✔️   
+Additional MSBuild-engine information | ❌        | ❌         | ❌         | ❌         | ✔️   
+
+In other words, you can think of the following log concepts per row as equivalent.
+
+NuGet log level    | MSBuild verbosity switch | MSBuild message type
+------------------ | ------------------------ | --------------------
+`Error`, `Warning` | `q[uiet]`                | Errors and Warnings
+`Minimal`          | `m[inimal]`              | High-importance Messages
+`Information`      | `n[ormal]`               | Normal-importance Messages
+`Verbose`, `Debug` | `d[etailed]`             | Low-importance Messages
+​                   | `diag[nostic]`           | Additional MSBuild-engine information
+
+### Example output
+
+This is output of the invocation of `dotnet nuget verify nuget.common.5.9.0-preview.2.nupkg`
+
+#### Default verbosity | `--verbosity mimimal`
+
+```
```

Comment:
What do you think of adding new lines between different sections? For example:

```
Verifying NuGet.Common.5.9.0-preview.2

Signature type: Author
Signing certificate:
  SHA256 hash: 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
  Valid from: 2/25/2018 4:00:00 PM to 1/27/2021 4:00:00 AM
Timestamp: 11/17/2020 7:23:10 AM
Timestamping service certificate:
  SHA256 hash: C474CE76007D02394E0DA5E4DE7C14C680F9E282013CFEF653EF5DB71FDF61F8
  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM

Signature type: Repository
nuget-v3-service-index-url: https://api.nuget.org/v3/index.json
nuget-package-owners: Microsoft, nuget
Signing certificate:
  SHA256 hash: 0E5F38F57DC1BCC806D8494F4F90FBCEDD988B46760709CBEEC6F4219AA6157D
  Valid from: 4/9/2018 5:00:00 PM to 4/14/2021 5:00:00 AM
Timestamp: 12/9/2020 2:32:12 PM
Timestamping service certificate:
  SHA256 hash: C474CE76007D02394E0DA5E4DE7C14C680F9E282013CFEF653EF5DB71FDF61F8
  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM

Successfully verified package 'NuGet.Common.5.9.0-preview.2'.
```

Reply:
I like the idea of separating out the sections as it improves readability.